Updates based on feedback from first three ELIPs. (#3)
eigenfoundation/ELIPs: template.md

|-------------|-----------|---------|------|----------|
| [Name(s) and contact information] | [Date in YYYY-MM-DD format] | [Draft/Review/Final] | [List of relevant work/PRs, if any] | [Link to discussion forum or thread] |

> *Note: This template may change slightly to suit the purpose and clarity of the proposal.*

Reviewer: ```suggestion
*Note: This template may change slightly to suit the purpose and scope of the proposal.*
```

Reviewer: @non-fungible-nelson - feel free to merge or disregard

## Executive Summary
*[A brief overview of the proposal, its main features, and expected impact]*

## Motivation
*[Explain why this proposal is necessary]*

## Features & Specification

> *[Introduction of any new features with detailed technical specifications for the upgrade]*

Reviewer: ```suggestion
*[Introduction of any new features including detailed technical specifications for proposed upgrade]*
```

Reviewer: @non-fungible-nelson - feel free to merge or disregard
